backend/test_observer/controllers/test_executions/models.py

```diff
@@ -54,6 +54,7 @@ class StartTestExecutionRequest(BaseModel):
     execution_stage: str
     environment: str
     ci_link: Annotated[str, HttpUrl]
+    test_plan: str = Field(max_length=200)
```

Comment:
Do you maybe know if other teams using Test Observer would be providing and using `test_plan`, I wonder if it is a better approach to make this optional and make it default to `""` in case not all teams are providing it, instead of making them explicitly call the API with `"test_plan": ""`. What do you think?

Reply:
I'd prefer if teams did specify some kind of string here. This is more important in the case of multiple teams running tests on the same environment. We'd need to specify which tests is for which team. We can communicate it to them as this is just a free form string in which they set a name for that kinds of tests they're running

Reply:
Fair enough, in that case we can keep the validation in place.